packages/app-store/caldavcalendar/api/add.ts

```diff
@@ -32,7 +32,6 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
 
     try {
       const dav = new CalendarService({
-        id: 0,
```

Comment:
This had to go since probably was put there to comply with type check

Reply:
Let's keep this PR as a mere hotfix. We might need to revert the migration so we don't have to deal with typings. Let's keep this fix simple.

Reply:
reverting

Reply:
It was working well before but it just showed me that the type it's not open to changes as it should be.